packages/vc-delegation-engine/tests/unit/jsonld-utils.test.js

```diff
@@ -0,0 +1,64 @@
+import { describe, it, expect } from 'vitest';
+
+import {
+  normalizeContextMap,
+  shortenTerm,
+  extractExpandedPresentationNode,
+  extractExpandedVerificationMethod,
+  findExpandedTermId,
+  normalizeSubject,
+  matchesType,
+} from '../../src/jsonld-utils.js';
+import { VC_TYPE, SECURITY_PROOF, SECURITY_VERIFICATION_METHOD } from '../../src/constants.js';
+
+const SUBJECT_ID = 'did:test:subject';
+
+describe('jsonld-utils', () => {
+  it('normalizes context maps from plain objects', () => {
+    const map = normalizeContextMap({ a: 1 });
+    expect(map).toBeInstanceOf(Map);
+    expect(map.get('a')).toBe(1);
+  });
+
+  it('shortens IRIs using hash, slash, or colon', () => {
```

Comment:
wasn't there a code path for pulling everything after the last slash? Shouldn't there be a test case here?